mne/epochs.py

```diff
@@ -779,10 +779,19 @@ class Epochs(_BaseEpochs, ToDataFrameMixin):
         Return Epochs object with a subset of epochs corresponding to an
         experimental condition as specified by 'name'.
 
+        If conditions are hierarchical (separated by '/', e.g. 'audio/left',
+        'audio/right') and 'name' is not in itself an event key, selects every
+        event whose condition contains 'name' (e.g., 'left' matches
+        'audio/left' and 'visual/left'; but not 'audio_left').
+
     epochs[['name_1', 'name_2', ... ]] : Epochs
         Return Epochs object with a subset of epochs corresponding to multiple
         experimental conditions as specified by 'name_1', 'name_2', ... .
 
+        If conditions are hierarchical, selects every item containing every
+        list item (e.g. ['audio', 'left'] selects 'audio/left' and
+        'audio/center/left', but not 'audio/right').
```

Comment:
In this sense you can just say that a list means an AND operation, right? I think that's more explicit.